VespaGAS/Managers/OilMixCalculator.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+let MLInLiter: Float = 1000
+
+typealias Oil = Volume
+typealias Gasoline = Volume
+
+struct OilMixCalculator {
+    static func oilFor(gasoline: Gasoline, oilMix: OilMix) -> Oil {
```

Comment:
you still have a static method here, how about moving this further into your domain?  

in TDD it is very easy to end up with what you have here which is called the [Anaemic domain model](http://www.martinfowler.com/bliki/AnemicDomainModel.html), where your domain objects are just bunches of data and you end up with static functions like this which do all the work. It is really worth trying to get around this because it makes the code far more approachable to someone coming on to a project and therefore reduces maintainence overhead 😸 it also means your code matches the domain more closely.

If you think of a concrete example - a new Java developer won't know how to square root a number. They'd probably expect something like `1.sqrt()` but instead they have to Google and find that it is instead `Math.sqrt(1)`!